lib/elixir/lib/system.ex

```diff
@@ -44,7 +44,7 @@ defmodule System do
       time may not match in case of time warps although the VM works towards
       aligning them. This time is not monotonic (i.e., it may decrease)
       as its behaviour is configured [by the VM time warp
-      mode](http://erlang.org/doc/apps/erts/time_correction.html#Time_Warp_Modes);
+      mode](http://www.erlang.org/doc/apps/erts/time_correction.html#Time_Warp_Modes);
```

Comment:
yes. links have all been standardized before. it's there because I have been recently introduced.
by standardized I mean, we use www.erlang.org, and we always use https:// when available (ie. Wikipedia, GitHub, etc)